### DISCUSSION TITLE
Minimint on the RPi4

### DISCUSSION THREAD

**Discussion by openoms** [raspiblitz/raspiblitz#3920] [Feature / App Ideas]
Getting [Minimint ](https://github.com/fedimint/minimint#minimint) running on a Raspiblitz would be very interesting and also contribute to this research issue: https://github.com/fedimint/minimint/issues/149

1. manual install guide
2. install script
3. implement as an optional service

**Comment by EthnTuttle** [comment]
Although it's not called Minimint anymore, I wanted to comment here that I aim to do this. 

There are two distinct daemons that could run on raspiblitz, fedimintd and gatewayd. Fedimintd is for someone wanting to be a fedimint "guardian" and use their server to provide consensus for the users. Gatewayd is how a LN node operator can provide services to a fedimint. It's allows the LN node operator to interact with LN/ecash swaps aka contracts.
